modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -239,7 +239,8 @@ class Instrument_Manager extends \NDB_Menu_Filter
                 . "Either: the instrument table exists (but is not in the "
                 . "test_names table), or "
                 . "LORIS could not connect to the database using the "
-                . "credentials supplied in the config file."
+                . "credentials supplied in the config file. "
+                . "Exec output: `" . implode("\n", $output) . "`"
```

Comment:
is there a risk the password could be printed out here ?

Reply:
It could be seen in the running process list while it is running. (top, ps, ...)

Reply:
It prints only the output of exec, in this case a MYSQL error code. It doesn't print the output of the command.

It might be possible to use `top` etc. to view it but this has been true since at least 2012 when the exec code above was written.